stable/pachyderm/templates/pachd_svc.yaml

```diff
@@ -2,67 +2,38 @@
 kind: Service
 apiVersion: v1
 metadata:
-  name: pachd-api-grpc
-  {{- if .Values.pachd.service.grpc.annotations }}
+  name: pachd
+  {{- if .Values.pachd.service.annotations }}
   annotations:
-    {{ .Values.pachd.service.grpc.annotations | toYaml | indent 4 | trim  }}
+    {{ .Values.pachd.service.annotations | toYaml | indent 4 | trim  }}
```

Comment:
`{{ toYaml .Values.pachd.service.annotations | indent 4 }}` could be make sense right?